src/components/flash/flash.js

```diff
@@ -223,19 +233,23 @@ class Flash extends React.Component {
    * @param {Object} ev
    * @return(Void)
    */
-  toggleDialog = (key, ev) => {
-    if (ev) { ev.preventDefault(); }
-
-    let state = this.state.dialogs[key];
-    // open/close the dialog
-    this.setState({ dialogs: { [key]: !state }});
-
-    // start/stop the timer if the dialog opens or closes
-    if (state) {
-      this.startTimeout();
-    } else {
-      this.stopTimeout();
-    }
+  toggleDialog = (key) => {
```

Comment:
Can you explain the change made here? If it is to do with the removal of the bind can we move the binding into the constructor or does the linter complain about that too? Im hoping that it doesn't